src/main/java/training/subsystems/Elbow/Constants.java

```diff
@@ -0,0 +1,18 @@
+package training.subsystems.Elbow;
+
+import edu.wpi.first.math.controller.ElevatorFeedforward;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public class Constants {
```

Comment:
if class can be protected i think it should be

Reply:
same in all places

Reply:
There is a solution, find it:)

Reply:
fixed; https://github.com/GreenBlitz/trainning-2024/commit/10a99810a348ce9d062269aa663404caf13c8a8c